Added custom entity data parameters (#2007)
MCreator/MCreator: plugins/mcreator-localization/help/default/entity/data_name.md

> This parameter defines the name of this variable.

Reviewer: Данный параметр задаёт имя этой переменной.
Используется в процедурах для изменения её значения.

> This text is the name of the variable.
> It is mainly used with procedure blocks to get or set a value.

Reviewer: ```suggestion
This parameter defines the name of this variable.
Mainly used with procedure blocks to get or set its value.
```
